spec/5-cross-cutting-requirements.md

```diff
@@ -1,7 +1,47 @@
 # 5 Cross-Cutting Requirements
-
+<!--
```

Comment:
I don’t actually know how commenting out these sections will be interpreted by the GitBook md interpreter. I guess we will find out!